apps/btshell/src/cmd.c

```diff
@@ -1752,6 +1752,15 @@ cmd_set_adv_data_or_scan_rsp(int argc, char **argv, bool scan_rsp)
         return rc;
     }
 
+    int8_t measured_power;
+    uint8_t measured_power_uint = parse_arg_uint8_dflt("measured_power", 0, rc);
+    if(rc == 0){
+        measured_power = -1 * measured_power_uint;
+    }else if(rc != ENOENT){
+        console_printf("invalid 'measured_power' parameter\n");
+        return rc;
+    }
```

Comment:
there are plenty of coding style issues in this small piece of code - please check `CODING_STANDARDS.md` file in repository and fix everywhere

Reply:
So you want me to send another PR with all the changes in one commit ? or would this do for now ?
I'll make changes wrt coding standards.

Reply:
just fix and cleanup commits on your local branch and force push to the same branch on github to update PR